### PR TITLE
REFACTOR: New edb cfg example dcir

### PR DESCRIPTION
**Pull request by hui-zhou-a**

### CODE REVIEW

File: examples/use_configuration/pcb_dc_ir.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    show_region=True,
+    show_axis=True,
+    show_grid=True,
+    show_ruler=True,
+)
+
+# ## Compute power loss
+
+p_layers = h3d.post.compute_power_by_layer(layers=["1_Top"])
+display(p_layers)
```
Comment (SMoraisAnsys) [inline examples/use_configuration/pcb_dc_ir.py:164]: @hui-zhou-a  Is there a reason to use `IPython.display` here ? Don't we have internal method to display the layers ?
Note that the associated image is note displayed in the HTML pages.

File: examples/use_configuration/pcb_dc_ir.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+from IPython.display import display
+from ansys.aedt.core import Hfss3dLayout
+from ansys.aedt.core.downloads import download_file
+
+from pyedb import Edb
+
+AEDT_VERSION = "2024.2"
+NG_MODE = False
+
+# -
```
Comment (SMoraisAnsys) [inline examples/use_configuration/pcb_dc_ir.py:22]:
```suggestion
# +
import json
import os
import tempfile

from IPython.display import display
from ansys.aedt.core import Hfss3dLayout
from ansys.aedt.core.downloads import download_file

from pyedb import Edb

AEDT_VERSION = "2024.2"
NG_MODE = False

# -
```

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, I left minor comments

File: examples/use_configuration/pcb_dc_ir.py (hunk 1 above)
Comment (hui-zhou-a) [reply examples/use_configuration/pcb_dc_ir.py:164]: @SMoraisAnsys could you give me an example of the internal method?

Comment (hui-zhou-a) [reply examples/use_configuration/pcb_dc_ir.py:163]: @SMoraisAnsys how to show a figure then?

Comment (hui-zhou-a) [reply examples/use_configuration/pcb_dc_ir.py]: I think it is an issue in PyAEDT. It doesn't plot automatically. I will investigate it in another branch.

**Review by SMoraisAnsys — DISMISSED** [review]
LGTM, I left a minor comment

**Review by svandenb-dev — APPROVED** [review]
LGTM

**Pull request merged**